main.py

```diff
@@ -1,10 +1,61 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        info = json.load(file)
+
+    for player_nickname, player_info in info.items():
+        race = create_race(player_info)
+        guild = create_guild(player_info)
+
+        Player.objects.create(
+            nickname=player_nickname,
+            email=player_info["email"],
+            bio=player_info["bio"],
+            race=race,
+            guild=guild
+        )
+
+        create_skills(player_info)
+
+
+def create_race(player_info: dict) -> Race:
+    if not Race.objects.filter(name=player_info["race"]["name"]).exists():
+        Race.objects.create(
+            name=player_info["race"]["name"],
+            description=player_info["race"]["description"]
+        )
+
+    return Race.objects.get(name=player_info["race"]["name"])
```

Comment:
can you optimize it? At least, you can make `race = Race.objects.create()...`. 
Then, you don't have to make an additional request to the database to get that value 
Now the difference is not felt much, but in real projects, it is VERY important.